### PR TITLE
docs: Update recommendation for usage in GitLab CI

### PR DESCRIPTION
**Pull request by underyx**
This follows the changes of https://github.com/returntocorp/semgrep-action/pull/19

Closes https://github.com/returntocorp/enterprise/issues/225

### CODE REVIEW

File: docs/integrations.md [hunk 1]
```diff
@@ -128,14 +128,13 @@ For more information on the GitHub Action see https://github.com/marketplace/act
...
-stages:
-    - test
-test:
-    image: python
-    before_script:
-        - python -m pip install semgrep
-    script:
-        - semgrep --config https://semgrep.live/p/r2c /path/to/code
+include:
+  - template: 'Workflows/MergeRequest-Pipelines.gitlab-ci.yml'
```
Comment (mschwager) [inline docs/integrations.md:132]: Is this needed for diff-awareness?

**Review by mschwager — COMMENTED** [review]
Should we update all the examples to recommend `semgrep_agent`?

**Comment by underyx** [comment]
> Should we update all the examples to recommend `semgrep_agent`?

No, because semgrep-agent needs to understand the CI environment to support diff awareness. This is only done for github actions and gitlab ci thus far.

File: docs/integrations.md (hunk 1 above)
Comment (underyx) [reply docs/integrations.md:132]: Any way of configuring merge request based pipelines is okay, this seems to be the easiest. If you don't take care, it's really easy to accidentally run on pushes instead as that's the default behavior, which is why I included this in the example.

**Review by brendongo — APPROVED** [review]

**Pull request merged**